processors/base/normalizers/phase_normalizer.py

```diff
@@ -0,0 +1,58 @@
+import os
+import csv
+import logging
+
+logger = logging.getLogger(__name__)
+
+def build_phase_variation_map(file):
+    """
+        Receives a path as an input and returns a dict containing
+        all normalized phases and their variations
+
+        :param
+            file (str): path to the phase variations file
+        :return:
+            variation map (dict): dict that contains all normalized phases and
+                                    their variations
+    """
+    variation_map = {}
+    with open(file, 'r') as variations:
+        reader = csv.reader(variations, quotechar='\'', delimiter=',')
+        for line in reader:
+            variation, target = line[0], line[1]
+            if target in variation_map.keys():
+                variation_map[target].append(variation)
+            else:
+                variation_map[target] = [target, variation]
+    return variation_map
+
+def get_normalized_phase(phase):
+    """ Receives a phase as an input and normalizes it if possible.
+        Else, returns the unormalized phase.
+
+        :param:
+            phase (str): unormalized phase
+
+        :return:
+            phase_suggestions (list): normalized phase suggestions
+    """
+    if not phase:
+        logger.debug('Unsuccessfully phase normalization \'None\'')
```

Comment:
Remember that `phase` can be any falsy value, like `[]`, `()`, `0`, not only `None`, so it's better to log the `phase` value itself.